tensorflow/compiler/xla/service/gpu/cudnn_fused_conv_rewriter_test.cc

```diff
@@ -239,19 +250,19 @@ TEST_F(CudnnFusedConvRewriterTest, TestScaledConvAndScaledSideInputWithBias) {
 }
 
 TEST_F(CudnnFusedConvRewriterTest, TestMatchMaxZeroOnly) {
-  // max(0.1, conv(x, w)) shouldn't match.
+  // max(1, conv(x, w)) shouldn't match.
```

Comment:
Change 0.1 to 1 won't affect the floating-point test cases.  Otherwise, I will make the change only for the s8 test case.